<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>高速视频下载器</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{border-image: url(:/images/mainBackgound.png);}QPushButton:menu-indicator{image:None;}</string>
   </property>
   <widget class="QToolButton" name="videoDownloadBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>318</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/videoDownload.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
   QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/videoDownload_Pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="screenBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>504</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/screen.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
   QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/screen_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="fileBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>504</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/file.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
   QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/file_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="netBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>504</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/net.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
  QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/net_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="logBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>846</x>
      <y>504</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/log.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
   QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/log_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>846</x>
      <y>318</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/tool.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
QToolButton:hover{border-width:2px;}
QToolButton:pressed{background-image: url(:/images/tool_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="diskBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>482</x>
      <y>318</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/disk.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
   QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/disk_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="shareBtn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>664</x>
      <y>318</y>
      <width>165</width>
      <height>165</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>165</width>
      <height>165</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>仿宋_GB2312</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{background-image: url(:/images/share.png);border-width:0px; border-radius: 5;border-style:solid; border-color: white;}
  QToolButton:hover{border-width:2px;}
   QToolButton:pressed{background-image: url(:/images/share_pressed.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>8</y>
      <width>560</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/logo.png)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBtn">
    <property name="geometry">
     <rect>
      <x>1230</x>
      <y>10</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>关机</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/shutdown.png</normaloff>:/images/shutdown.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>1186</x>
      <y>10</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Min</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/min.png</normaloff>:/images/min.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="moreBtn">
    <property name="geometry">
     <rect>
      <x>1143</x>
      <y>10</y>
      <width>45</width>
      <height>45</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>More</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="fastvideo.qrc">
      <normaloff>:/images/more.png</normaloff>:/images/more.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelProduct">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>242</y>
      <width>707</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  	font-size:40px;
	color: rgb(255, 255, 255);
	font.family: &quot;楷体&quot;;
}</string>
    </property>
    <property name="text">
     <string>视频高速下载器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
